src/presence_analyzer/static/css/main.css

```diff
@@ -0,0 +1,61 @@
+body {
```

Comment:
Usually the way to name a main template / stylesheet is `base.css`, `main` can be mistaken with main page.